client/src/app/tabs/dmn/DmnEditor.js

```diff
@@ -908,3 +1004,32 @@ function getMigrationDialog() {
     checkboxLabel: 'Do not ask again.'
   };
 }
+
+/**
+ * Check layout whether overview is open.
+ *
+ * @param {Object} props
+ *
+ * @returns {boolean}
+ */
+function isOverviewOpen(props) {
+  return props.layout && props.layout.dmnOverview && props.layout.dmnOverview.open;
+}
+
+/**
+ * Check layout whether overview was opened.
+ *
+ * @param {Object} props
+ * @param {Object} prevProps
+ *
+ * @returns {boolean}
+ */
+function isOverviewOpenend(props, prevProps) {
```

Comment:
Typo in the method name which is by the way really similar to `isOverviewOpen`. It led to my confusion so how about if we change it to idk `isOpeningOverview`?

Reply:
Good catch. I thought `isOverviewOpened` would be a good name because it compares previous to current props, whereas `isOverviewOpen` looks at current props. So it's not really opening from the props perspective since they already have changed.